frontend/cypress/integration/common/workload_logs.ts

```diff
@@ -1,88 +1,96 @@
 import { Given, Then, When } from '@badeball/cypress-cucumber-preprocessor';
 
-Given('I am on the {string} workload detail page of the {string} namespace', (workload, namespace) => {
+Given('I am on the {string} workload detail page of the {string} namespace', (workload: string, namespace: string) => {
   cy.visit(`/console/namespaces/${namespace}/workloads/${workload}?refresh=0`);
 });
 
-Given('I am on the logs tab of the {string} workload detail page of the {string} namespace', (workload, namespace) => {
-  cy.visit(`/console/namespaces/${namespace}/workloads/${workload}?tab=logs&refresh=0`);
-});
+Given(
+  'I am on the logs tab of the {string} workload detail page of the {string} namespace',
+  (workload: string, namespace: string) => {
+    cy.visit(`/console/namespaces/${namespace}/workloads/${workload}?tab=logs&refresh=0`);
+  }
+);
 
 When('I go to the Logs tab of the workload detail page', () => {
   cy.get('[data-test=workload-details-logs-tab]').click();
 });
 
-When('I type {string} on the Show text field', showText => {
-  cy.get('#log_show').type(showText + '{enter}');
+When('I type {string} on the Show text field', (showText: string) => {
+  cy.get('#log_show').type(`${showText}{enter}`);
 });
 
-When('I type {string} on the Hide text field', showText => {
-  cy.get('#log_hide').type(showText + '{enter}');
+When('I type {string} on the Hide text field', (showText: string) => {
+  cy.get('#log_hide').type(`${showText}{enter}`);
 });
 
 When('I choose to show 100 lines of logs', () => {
   cy.get('#wpl_maxLines-toggle').click();
   cy.get('#100').click();
 });
 
-When('I select only the {string} container', containerName => {
+When('I select only the {string} container', (containerName: string) => {
   cy.get('[data-test=workload-logs-pod-containers]').within(() => {
-    cy.get('[type=checkbox]:checked').uncheck();
+    cy.get('[type=checkbox]').uncheck();
+
     cy.get(`input#container-${containerName}`).check();
   });
 });
 
 When('I enable visualization of spans', () => {
   cy.get('[data-test=workload-logs-pod-containers]').within(() => {
-    cy.get('[type=checkbox]:checked').uncheck();
```

Comment:
This is a different assertion now?

Reply:
Previously only checked checkboxes were unchecked. Now all checkboxes are unchecked, for the cypress test purpose is the same...

This was the Cypress error:
```
CypressError: Timed out retrying after 40100ms: `cy.uncheck()` failed because the page updated while this command was executing. Cypress tried to locate elements based on this query:
```
I think the system gets unstable because unchecks are run in parallel, so it is hard for cypress to determine which checkboxes are checked if at the same time some of them are being unchecked.